python_functionality/homework.py

```diff
@@ -35,21 +42,26 @@ def task_3_find_item_via_value(data: DT, value) -> DT:
         find_item_via_value([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 26)
         >>> [{'name': 'Alex', 'age': 26}]
     """
-    pass
+    items_finded = []
+    for dic in data:
+        for key in dic:
+            if value == dic[key]:
+                items_finded.append(dic)
```

Comment:
also maybe here you would use list comp?

Reply:
Now looks more laconic and expressive